sky/engine/core/script/dart_init.cc

```diff
@@ -86,10 +86,6 @@ static const char* kDartCheckedModeArgs[] = {
     "--error_on_bad_override",
 };
 
-void UnhandledExceptionCallback(Dart_Handle error) {
-  LOG(ERROR) << Dart_GetError(error);
-}
```

Comment:
Yes. The Dart API no longer supports the embedder providing this argument via Dart_Initialize. Will follow up on if there is now an alternative. Basically, that argument has to be nil. See https://github.com/dart-lang/sdk/blob/master/runtime/vm/dart_api_impl.cc#L1131